projects/plugins/jetpack/modules/widgets/simple-payments/form.php

```diff
@@ -17,7 +18,12 @@ class="widefat jetpack-simple-payments-widget-title"
 		name="<?php echo esc_attr( $this->get_field_name( 'title' ) ); ?>"
 		value="<?php echo esc_attr( $instance['title'] ); ?>" />
 </p>
-<p class="jetpack-simple-payments-products-fieldset" <?php if ( empty( $product_posts ) ) { echo 'style="display:none;"'; } ?>>
+<p class="jetpack-simple-payments-products-fieldset" 
+<?php
+if ( empty( $product_posts ) ) {
+	echo 'style="display:none;"'; }
```

Comment:
```suggestion
	echo 'style="display:none;"';
}
```

Reply:
What the heck happened with all these brackets?

Reply:
Apparently we have a sniff enabled for "code should not follow the open-brace", but not one for "close-brace should be on its own line too". So the former got fixed while the latter was never flagged.